### PR TITLE
add get reviews/, add testing (with jest-sorted)

### PR DESCRIPTION
**Pull request by xViolaine**
I have added a get reviews endpoint which allows us to get an array of review objects, they are sorted by date in descending order.
I have also added a test for this endpoint that utilizes jest-sorted.

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -53,6 +53,33 @@ describe("GET /api/users", () => {
...
+            .get('/api/reviews/')
+            .expect(200)
+            .then(({ body }) => {
+                const { reviews } = body;
+                expect( reviews ).toBeInstanceOf(Array);
+                expect(reviews).toHaveLength(13);
+                expect(reviews).toBeSortedBy('created_at', { descending: true });
+                reviews.forEach((review) => {
+                    expect(review).toEqual(
+                        expect.objectContaining({
```
Comment (dappernerddesigns) [inline __tests__/app.test.js]: Is this strictly testing that a review body has NOT been returned? ojectContaining may give you a false positive here.

File: models/models-games.js [hunk 2]
```diff
@@ -32,3 +30,15 @@ exports.selectUsers = () => {
...
 };
+
+exports.selectReviews = () => {
+    return db.query(`SELECT reviews.owner, reviews.review_id, reviews.created_at, reviews.title, reviews.category, reviews.review_img_url, reviews.votes, CAST (COUNT(comments) AS INTEGER) AS comment_count
+    FROM reviews
+    LEFT JOIN comments
+    ON reviews.review_id = comments.review_id
+    GROUP BY reviews.review_id
+    ORDER BY created_at DESC`).then((reviews) => {
+        console.log(reviews.rows)
```
Comment (dappernerddesigns) [inline models/models-games.js:41]: Make sure not to leave console logs that we don't need :)

File: __tests__/app.test.js (hunk 1 above)
Comment (dappernerddesigns) [reply __tests__/app.test.js]: While I can see in your model you are not asking for the body, could you make the test more robust?

**Review by dappernerddesigns — CHANGES_REQUESTED** [review]
Some tiny changes to consider. Very tidy so far though.

File: models/models-games.js (hunk 2 above)
Comment (xViolaine) [reply models/models-games.js:41]: Thanks, I knew I forgot something :D

File: __tests__/app.test.js (hunk 1 above)
Comment (xViolaine) [reply __tests__/app.test.js]: I'll give it a try!

**Pull request merged**